Vagrantfile

```diff
@@ -11,6 +11,7 @@ Vagrant.configure("2") do |config|
     override.vm.box_version = "1.0.0"
     # docker (in-docker) needs privileges for creating the docker socket
     docker.create_args = [ "--cap-add=NET_ADMIN" ]
+    docker.create_args = [ "--env='DISPLAY=host.docker.internal:0'", "-v", "/tmp/.X11-unix:/tmp/.X11-unix:rw" ]
```

Comment:
Note: passing the `DISPLAY` env var does not work here. It seems to be passed correctly to the container, but then [when starting sshd](https://github.com/tknerr/vagrant-docker-baseimages/blob/master/ubuntu-2004/Dockerfile#L43-L47) in the container the current user environment is ([for good reasons](https://unix.stackexchange.com/a/573938)) not being passed to sshd.

So instead, we should rather set DISPLAY conditionally (i.e. when running in a docker container) via bashrc.d